src/components/Modal.js

```diff
@@ -0,0 +1,37 @@
+import React, { forwardRef } from 'react';
+import ReactDom from 'react-dom';
+
+const modalStyles = {
+  position: 'fixed',
+  top: '50%',
+  left: '50%',
+  transform: 'translate(-50%,-50%',
+  backgroundColor: '#FFF',
+  padding: '40px',
+  zIndex: 1000,
+  width: 200,
+  borderRadius: '0.3rem',
+  textAlign: 'center',
+};
+
+const overlayStyles = {
+  position: 'fixed',
+  top: 0,
+  left: 0,
+  right: 0,
+  bottom: 0,
+  backgroundColor: 'rgba(0,0,0,.7)',
+  zIndex: 1000,
+};
+
+export default forwardRef(function Modal({ open, children, modalRef }) {
```

Comment:
`forwardRef` takes 2 props (props, ref) so the syntax here would need to be:
```suggestion
export default forwardRef(function Modal({ open, children}, modalRef) {
```

Reply:
That said, this doesn't quite get you where you want to be because the `current` property of the modal will not be available until after it renders and as such will likely always be `null` when called in the delete button component